pom.xml

```diff
@@ -292,8 +293,17 @@
         <mockito.version>1.10.19</mockito.version>
         <joda.version>2.2</joda.version>
         <metrics-elasticsearch-reporter.version>2.2.0</metrics-elasticsearch-reporter.version>
+
+        <common.cli.version>1.3.1</common.cli.version>
+
+        <!-- dropwizard -->
+        <dropwizard.version>0.7.1</dropwizard.version>
```

Comment:
It should be ok. As dropwizard is a more full stack framework than jersey. As already included, we we may directly use some other features of dropwizard as well.